### ISSUE TITLE
can not building lm.binary using unigram 

### ISSUE THREAD

**Issue by Nourahussein** [kpu/kenlm#182]
I am trying to use a unigram arpa file to build a kenlm Model in the python wrapper. However, I receive the following error:

![error](https://s33.postimg.cc/re2wgmu5r/Screenshot_from_2018-09-08_19-49-34.png)
is there any way to create a binary file using unigrams?

**Comment by kpu** [comment]
No, unigrams create corner cases (the highest order is the same as the lowest order) so I haven't implemented them.

**Comment by thuvh** [comment]
you can check #26

**Comment by lpcauch** [comment]
Hey @Nourahussein, 

Dis you succeed ? I'm stuck at the creation of the lm.binary, even by using the solution of @thuvh 
I created the unigram without any problem but when i wanted to create the lm.binary it throw this error "lm/read_arpa.cc:85 in void lm::ReadNGramHeader(util::FilePiece&, unsigned int) threw FormatLoadException'.
Was expecting n-gram header \1-grams: but got \end\ instead Byte: 209
ERROR"
any advices ? 
thanks !
